utils/quay_api.py

```diff
@@ -80,15 +76,12 @@ def remove_user(self, user):
 
         return True
 
-    def add_user_team(self, user):
-        if self.team is None:
-            raise('Must define "team"')
-
+    def add_user_to_team(self, user, team):
         if user in self.list_team_members(cache=True):
```

Comment:
I think you're missing `team` here:

```suggestion
        if user in self.list_team_members(team, cache=True):

Reply:
thanks!